### ISSUE TITLE
Multiple filters - logical behavior

### ISSUE THREAD

**Issue by bvialade** [aslagle/reactive-table#446]
When I create multiple own filters, using ReactiveTable.Filter, the reactive-table displayed is filled with a logical "AND" of filters (data is in the both filtered results).
Is there an alternative solution to get a "OR" result ? (data is at least in one of filtered result).

**Comment by aslagle** [comment]
Yes, you can put `filterOperator: "$or"` in the settings, either on the client or server depending on if you are using server-side filtering.

**Comment by bvialade** [comment]
Thanks !

**Comment by bvialade** [comment]
Bonus question:
Can we combine $or and $and filters operators in my filter list, or the filterOperator is set for all filters ?
Thanks,

**Comment by aslagle** [comment]
Sorry, no - it's one setting for all the filters.
